circuits/RayTracing10.circom

```diff
@@ -0,0 +1,5 @@
+pragma circom 2.0.0;
+
+include "./PointInPolygon.circom";
+
+component main = RayTracing(10, 26);
```

Comment:
Wonder if there's an intuitive way of merging all specific implementations for .circom and _input.json so we don't have a ton of these files. 

Just a nice to have.